plugins/CoreAdminHome/Controller.php

```diff
@@ -320,21 +320,34 @@ public function trackingCodeGenerator()
      */
     public function optOut()
     {
+        $reref = Common::getRequestVar('reref', false);
+        if (!empty($reref)) {
+            Url::redirectToUrl($reref);
```

Comment:
actually that's dead code from one of the other attempts at fixing it. It can be removed, thanks for catching it!